core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -239,26 +357,34 @@ object JsonAST {
       if (f.isDefinedAt(x)) f(x) else x
     }
 
-    /** Return a new JValue resulting from applying the given partial function <code>f</code>
-     * to each value in JSON.
-     * <p>
-     * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: Nil) transform { case JInt(x) => JInt(x+1) }
-      * </pre>
-      */
+    /**
+     * Return a new `JValue` resulting from applying the given partial function to each value.
```

Comment:
Each value in what? I think it may be worth enumerating `JObject` vs `JArray` behavior here as is done above, assuming it's different.